packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: v0.0.1
+description: Helm chart for installing the rancher clusterscan-operator than runs security scans on a kubernetes cluster
+name: rancher-cis-benchmark
+version: 0.0.1
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/namespace: cis-operator-system
```

Comment:
We aren't prepending the NS with cattle ? I know that was recommendation at one point, we can talk about it.

Reply:
In the feature channel it was actually suggested to not use cattle prefix - and also that UI is just looking for `-system ` suffix

Reply:
@prachidamle For our charts, let's make sure to prefix it with `cattle`

Reply:
Need to clarify with @vincent99 and @ibuildthecloud if we are still using "cattle-*-system" convention for namespace - or just '*-system' is good enough.